test/useTranslation.ready.spec.js

```diff
@@ -39,33 +41,45 @@ describe('useTranslation', () => {
     expect(() => {
       console.error = jest.fn(); // silent down the error boundary error from react-dom
 
-      mount(<TestComponentNotReady i18n={instance} />, {});
+      render(<TestComponentNotReady i18n={instance} />);
     }).toThrow(
       'TestComponentNotReady suspended while rendering, but no fallback UI was specified.',
     );
     expect(console.error).toHaveBeenCalled(); // silent down the error boundary error from react-dom
   });
 
   it('should render correct content if ready (having all ns)', () => {
-    const wrapper = mount(<TestComponent i18n={instance} />, {});
-    // console.log(wrapper.debug());
```

Comment:
I think we should keep such comments, but `// screen.debug()`

Reply:
interesting point - given that `toMatchInlineSnapshot` already provides a sufficient error message on failure (especially when compared to the previous `.toBe(true)`), I don't think that `// screen.debug()` is even necessary...

a failed test would show a nice diff between actual and expected HTML => no need to run the test again with uncommented line

Reply:
@Aprillion Hmm. Agreed :tada:, that is even logical :smiley:, thanks for your input, I should keep asking for other opinions :raised_hands: